mimic/resource.py

```diff
@@ -84,6 +91,14 @@ def advance_time(self, request):
             "now": seconds_to_timestamp(self.clock.seconds())
         })
 
+    @app.route("/mimic/v1.1/behaviors", branch=True)
```

Comment:
Sure, I can make it `IdentityControlAPI/behaviors` instead.

And this is v1.1 of the mimic API, as opposed to v2.0 of the identity API.  TBH I wasn't really sure where to put this,whether we wanted to put it under `/mimic/` or whether we wanted to make it `/identityControl`, like how identity is actually under `/identity`.